apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/DateTimestampReadTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright 2020 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    [Collection(nameof(DateTimestampTableFixture))]
+    [CommonTestDiagnostics]
+    public class DateTimestampReadTests
+    {
+        private readonly DateTimestampTableFixture _fixture;
+
+        public DateTimestampReadTests(DateTimestampTableFixture fixture) =>
+            _fixture = fixture;
+
+        [Fact]
+        public async void WriteDateThenRead_ShouldBeEqual()
+        {
+            using (var connection = _fixture.GetConnection())
+            {
+                foreach (var date in new DateTime[] {
```

Comment:
I think it would be easier to read if you declared a variable with these values in rather than declaring the array inline within the foreach statement itself.

Alternatively, remove the `DateTime.Today` value and use a parameterized test, which makes it easier to see which ones work. (I don't tend to like using `Today` or `Now` in tests anyway. They can lead to flaky tests if something only fails if the second is 0 for example.)

Reply:
(Same comment for the other test.)

Reply:
I've changed the tests to a `Theory` and removed the `Now` and `Today` values. The important test values here are the different kinds, and not the actual date values.